src/test/java/org/jenkinsci/plugins/tokenmacro/impl/ChangesSinceLastBuildMacroTest.java

```diff
@@ -358,4 +346,19 @@ public Collection<AffectedFile> getAffectedFiles() {
             };
         }                
     }
+
+    private AbstractBuild createBuild2(Result result, int buildNumber, String message) {
+        AbstractBuild build = mock(AbstractBuild.class);
+        ChangeLogSet changes1 = createChangeLog2(message);
+
```

Comment:
nit: extra whitespace here.

Reply:
Thanks for pointing it out, we removed it.